src/rust/ide/view/graph-editor/src/component/node/output/port.rs

```diff
@@ -365,35 +365,42 @@ impl display::Object for PortShapeView {
 
 ensogl::define_endpoints! {
     Input {
-        set_size_multiplier (f32),
-        set_definition_type (Option<Type>),
-        set_usage_type      (Option<Type>),
+        set_size_multiplier  (f32),
+        set_definition_type  (Option<Type>),
+        set_usage_type       (Option<Type>),
+        set_label_visibility (bool),
+        set_size             (Vector2),
     }
 
     Output {
         tp       (Option<Type>),
         on_hover (bool),
         on_press (),
-        tooltip  (tooltip::Style),
+        size     (Vector2),
     }
 }
 
 #[derive(Clone,Debug,Default)]
 pub struct Model {
-    pub frp    : Option<Frp>,
-    pub shape  : Option<PortShapeView>,
-    pub index  : usize,
-    pub length : usize,
+    pub frp            : Option<Frp>,
+    pub shape          : Option<PortShapeView>,
+    pub label          : Option<text::Area>,
```

Comment:
This is port model. So in port model we have a field label. Is this label of a port?